pkg/debug/scripts/collect-info.sh

```diff
@@ -216,6 +216,37 @@ collect_pillar_backtraces()
 
     echo "- done pillar backtraces"
 }
+collect_zfs_info()
+{
+        type=$(cat /run/eve.persist_type)
+        if [ "$type" = "zfs" ]; then
+           echo "- Collecting ZFS specific info"
+           echo "zpool status" >> "$DIR/zfs-info"
+           echo "============" >> "$DIR/zfs-info"
```

Comment:
I agree with shellcheck, you can avoid repeating `>> "$DIR/zfs-info"` on every line using a code block, like we do here: https://github.com/lf-edge/eve/blob/master/pkg/debug/scripts/collect-info.sh#L228-L233